opentelemetry-resource-detectors/Cargo.toml

```diff
@@ -16,3 +16,6 @@ rust-version = "1.70.0"
 opentelemetry = { workspace = true }
 opentelemetry_sdk = { workspace = true }
 opentelemetry-semantic-conventions = { workspace = true }
+
+[dev-dependencies]
+temp-env = "0.3.6"
```

Comment:
interesting...there were existing tests that used this somehow?

Reply:
yeah, they existed, but they were not exposed in the `lib.rs` and therefore never actually invoked 😅